### ISSUE TITLE
Copy/Paste with passenger entities

### ISSUE THREAD

**Issue by TerraMiner** [IntellectualSites/FastAsyncWorldEdit#2427]
### Server Implementation

Paper

### Server Version

1.19.4

### Describe the bug

When I copy 2 entities, one of which sits on the other, and try to paste it, the following error pops up, and the entity that sits is not pasted. It is also possible that this is due to the fact that entity 2 is ItemDisplay.

Error:
Unknown entity tag: BinaryTagType[CompoundBinaryTag 10]{tags={"UUIDMost"=BinaryTagType[LongBinaryTag 4 (numeric)]{value=7102332560571647860}, "Pos"=BinaryTagType[ListBinaryTag 9]{tags=[BinaryTagType[DoubleBinaryTag 6 (numeric)]{value=30.5d}, BinaryTagType[DoubleBinaryTag 6 (numeric)]{value=9.0d}, BinaryTagType[DoubleBinaryTag 6 (numeric)]{value=-3018.5d}], type=BinaryTagType[DoubleBinaryTag 6 (numeric)]}, "UUIDLeast"=BinaryTagType[LongBinaryTag 4 (numeric)]{value=-7375300107981949919}, "WorldUUIDMost"=BinaryTagType[LongBinaryTag 4 (numeric)]{value=7102332560571647860}, "PersistentIDLSB"=BinaryTagType[LongBinaryTag 4 (numeric)]{value=-7375300107981949919}, "PersistentIDMSB"=BinaryTagType[LongBinaryTag 4 (numeric)]{value=7102332560571647860}, "Id"=BinaryTagType[StringBinaryTag 8]{value="minecraft:item_display"}, "UUID"=BinaryTagType[IntArrayBinaryTag 11]{value=[1653640661, -2054142092, -1717195872, -1210715103]}, "WorldUUIDLeast"=BinaryTagType[LongBinaryTag 4 (numeric)]{value=-7375300107981949919}}}

### To Reproduce

select any territory with 2 passengers entities
write //copy -e
paste this with //paste -e
and see this error and bug

### Expected behaviour

Both entities with tags applied should be inserted.

### Fawe Debugpaste

https://athion.net/ISPaster/paste/view/bd703c0ec10d4689876eda5c18024f10

### Fawe Version

FastAsyncWorldEdit-Bukkit-2.7.2-SNAPSHOT-557.jar

### Checklist

- [X] I have included a Fawe debugpaste.
- [X] I am using the newest build from https://ci.athion.net/job/FastAsyncWorldEdit/ and the issue still persists.

**Comment by OneLiteFeather** [comment]
Please use Paper

**Comment by OneLiteFeather** [comment]
I'll look into reproducing it, also @TheMeinerLP (for your information, we talked about that)

**Comment by TerraMiner** [comment]
I can record a video of the bug with the console in the background if needed.

**Comment by TerraMiner** [comment]
I am not a regular user of this forum, but I would like to know how quickly this problem can be solved?

**Comment by OneLiteFeather** [comment]
> I am not a regular user of this forum, but I would like to know how quickly this problem can be solved?

Depends on how complicate this turns out. We had a similar issue fixed but somehow it wasn't 100% fixed I guess.

**Comment by TerraMiner** [comment]
https://youtu.be/Bw7aShMvKJ4

In the video you can see 2 entities, one of which is Interaction, the second Item Display. On the Interaction entity (Which you can see the hitbox), sits an entity of the Item Display type.

(Ignore the texture, the server is using a resource pack)

**Comment by TheMeinerLP** [comment]
> https://youtu.be/Bw7aShMvKJ4
> 
> In the video you can see 2 entities, one of which is Interaction, the second Item Display. On the Interaction entity (Which you can see the hitbox), sits an entity of the Item Display type.
> 
> (Ignore the texture, the server is using a resource pack)
> 
> Edited: I think that the error occurs in this place due to the absence of the Rotation tag. ![image](https://user-images.githubusercontent.com/29194071/269743433-9f74488b-f8fe-49e6-9aab-79c091ba7597.png)

Funny i don't see the original entities(makes very complicated) also the video is to short. The last 5~ frames shows the paste operation. But little bit to short to pause the video...

**Comment by TerraMiner** [comment]
I made a quick but ugly code solution that will solve the problem only in a very narrow range of situations (will only fix ItemDisplay)
https://pastebin.com/5S1UfDkY

**Comment by TheMeinerLP** [comment]
Thats not the right solution. My PR for copy entites was not merged yet. Also there improvements missing for 5 Stacked Entity

**Comment by TerraMiner** [comment]
Yes, I completely agree, but even this solution still satisfies the needs of builders who will work with a private plugin for fittings. Because they desperately needed a solution to this problem.
